core/src/main/java/org/codice/imaging/nitf/core/image/impl/RasterProductFormatUtilities.java

```diff
@@ -59,25 +66,42 @@ public RasterProductFormatUtilities() throws NitfFormatException {
     }
 
     /**
-        Unmarshal input stream to RPFs list.
-
-        @param inputStream the input stream to parse.
-        @throws JAXBException if the unmarshalling of the XML fails.
-    */
+     * Unmarshal input stream to RPFs list.
+     *
+     * @param inputStream the input stream to parse.
+     * @throws JAXBException if the unmarshalling of the XML fails.
+     */
     private void unmarshal(final InputStream inputStream) throws JAXBException {
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+
+        dbf.setExpandEntityReferences(false);
+
+        Document document;
+        try {
+            DocumentBuilder db = dbf.newDocumentBuilder();
+            document = db.parse(inputStream);
+        } catch (SAXException | IOException e) {
+            LOG.warn("Error parsing input. Set log to DEBUG for more information.");
+            LOG.debug("Error parsing input. {}", e);
+            throw new JAXBException(e);
+        } catch (ParserConfigurationException e) {
+            LOG.error("Error creating DocumentBuilder. Set log to DEBUG for more information.");
+            LOG.debug("Error creating DocumentBuilder. {}", e);
+            throw new JAXBException(e);
+        }
```

Comment:
Maybe that's the solution then, if we're sure this particular unmarshaller will never be susceptible to entity expansion, then we can probably consider this particular issue a "Not an issue"